configuration-ghc-921.nix

```diff
@@ -5,28 +5,43 @@ let
   disabledPlugins = [
     "hls-brittany-plugin"
     "hls-stylish-haskell-plugin"
+    "hls-hlint-plugin"
+    "hls-haddock-comments-plugin"
+    "hls-alternate-number-format-plugin"
+    "hls-eval-plugin"
+    "hls-tactics-plugin"
+    # That one is not technically a plugin, but by putting it in this list, we
+    # get it removed from the top level list of requirement and it is not pull
+    # in the nix shell.
+    "shake-bench"
```

Comment:
I think the class plugin should also be disabled. Is it buildable (at least it wasn't last time I tried to make it work with GHC9.2)?

Reply:
This list is mostly here to disable the pulling of dependencies which are broken. Considering that the shell and binaries can be built, I don't see the point of removing dependencies for `class` from the shell.